src/browser/modules/D3Visualization/lib/visualization/components/Visualization.ts

```diff
@@ -278,18 +278,40 @@ const vizFn = function (
     }
 
     function dragHandler(simulation: Simulation<VizNode, Relationship>) {
+      let initialPos: [number, number]
+      let restartedSim = false
+      const tolerance = 25
+
+      const dist = (a: [number, number], b: [number, number]) =>
+        Math.pow(a[0] - b[0], 2) + Math.pow(a[1] - b[1], 2)
```

Comment:
If we're going for euclidean distance we're missing `Math.sqrt()` here right? (this still kind of works though)

Reply:
Yeah, that's right. I took this from the code we had before and I removed the `Math.sqrt()`. I don't remember if the difference was noticeable, but the reasoning here was to remove unnecessary computation, since this function is run loads of times while dragging. I squared the tolerance too, so it is essentially the same.

Reply:
Ah neat. Perhaps that's what the previous implementation did as well. I "fixed" it and added the sqrt. perhaps we should rename it to distanceIsh or add a comment so others don't "fix" it again